Sources/Prometheus/Histogram.swift

```diff
@@ -67,20 +55,29 @@ public final class Histogram<Value: Bucketable>: Sendable {
             state.count += 1
         }
     }
+
+    public func observe(_ value: Duration) {
```

Comment:
So we could record durations into a histogram intended for values; this seems silly for our ecosystem in which we value type safety for such stuff.